### PR TITLE
Different warning flag if all warnings confirmed

### LINKED ISSUE #923: Different warning flag if all warnings confirmed
If all warnings are confirmed the flag in the list of patient should be different than if warnings are unconfirmed

### PR DESCRIPTION
**Pull request by kristinagalik**
Closes #923

### CODE REVIEW

**Review by kubantjan — CHANGES_REQUESTED** [review]
One general point. I really thing we need to have something that would distinguish the patients that are included in the matching, Something that on the first glance shows which patients are included and which not. 
I would do two things:
1) In the list of patients sent to FE first send the patients that are included and then the ones that are not (including the inactive patients). 
2) As we have the inactive label than have similar label in case the patient is not inactive but is not included (in this current case it is patients with errors and patients with unconfirmed warnings) The lable could be simply  something like "excluded"

The confirmed/unconfirmed warning distinction is ok, but maybe i would prefer something more clear (making the confirmed warnings less threatening - maybe green even? as they are not of concern at all)

I would skip senindg patient_has_Confirmed_warnings to FE you could simply check there if all warnings are confirmed or not..

**Review by kubantjan — APPROVED** [review]
za me ok, jeste @NotLubo

File: txmatching/web/frontend/src/app/components/patient-pair-item/patient-pair-item.component.ts [hunk 1]
```diff
@@ -22,5 +23,8 @@ export class PatientPairItemComponent extends ListItemAbstractComponent implemen
   ngOnInit() {
     this.allMessagesCount = countAllMessages(this.item?.d) + countAllMessages(this.item?.r);
     this.mostSevereMessageType = findMostSevereMessageType(this.item?.d, this.item?.r);
+    if (patientHasConfirmedWarnings(this.item?.d?.allMessages) && patientHasConfirmedWarnings(this.item?.r?.allMessages)){
```
Comment (NotLubo) [inline txmatching/web/frontend/src/app/components/patient-pair-item/patient-pair-item.component.ts:26]: if není potřeba
` this.allWarningsConfirmed = patientHasConfirmedWarnings(this.item?.d?.allMessages) && patientHasConfirmedWarnings(this.item?.r?.allMessages);`

**Review by NotLubo — CHANGES_REQUESTED** [review]
Komentáře

File: txmatching/web/frontend/src/app/components/patient-pair-item/patient-pair-item.component.ts [hunk 2]
```diff
@@ -22,5 +23,9 @@ export class PatientPairItemComponent extends ListItemAbstractComponent implemen
   ngOnInit() {
     this.allMessagesCount = countAllMessages(this.item?.d) + countAllMessages(this.item?.r);
     this.mostSevereMessageType = findMostSevereMessageType(this.item?.d, this.item?.r);
+    this.allWarningsConfirmed = patientHasConfirmedWarnings(this.item?.d?.allMessages) &&
+                                patientHasConfirmedWarnings(this.item?.r?.allMessages);
   }
 }
+
+
```
Comment (NotLubo) [inline txmatching/web/frontend/src/app/components/patient-pair-item/patient-pair-item.component.ts:31]: 2 prázdne řádky. jeden odstranit

Comment (NotLubo) [inline txmatching/web/frontend/src/app/components/patient-pair-item/patient-pair-item.component.ts:31]: 2 prázdne řádky jeden odstranit

File: txmatching/web/frontend/src/app/parsers/parsingIssueConfirmation.parsers.ts [hunk 3]
```diff
@@ -3,6 +3,14 @@ import { ParsingIssueConfirmation } from '../model';
...
-    ...data
+    confirmedAt: data.confirmed_at,
+    confirmedBy: data.confirmed_by,
+    dbId: data.db_id,
+    donorId: data.donor_id,
+    hlaCodeOrGroup: data.hla_code_or_group,
+    message: data.message,
+    parsingIssueDetail: data.parsing_issue_detail,
+    recipientId: data.recipient_id,
+    txmEventId: data.txm_event_id,
```
Comment (NotLubo) [inline txmatching/web/frontend/src/app/parsers/parsingIssueConfirmation.parsers.ts:14]: ESLint hlási trailing comma

**Review by NotLubo — CHANGES_REQUESTED** [review]
Formátování. Zišlo by se do projektu přidat prettier

**Review by NotLubo — APPROVED** [review]
Super za mňa ok :)

**Pull request merged**